forms/account-freemarker/src/main/java/org/keycloak/account/freemarker/FreeMarkerAccountProvider.java

```diff
@@ -197,7 +197,11 @@ public Response createResponse(AccountPages page) {
             String result = freeMarker.processTemplate(attributes, Templates.getTemplate(page), theme);
             Response.ResponseBuilder builder = Response.status(status).type(MediaType.TEXT_HTML).entity(result);
             BrowserSecurityHeaderSetup.headers(builder, realm);
-            LocaleHelper.updateLocaleCookie(builder, locale, realm, uriInfo, Urls.localeCookiePath(baseUri,realm.getName()));
+
+            String keycloakLocaleCookiePath = Urls.localeCookiePath(baseUri, realm.getName());
```

Comment:
Actually thinking about this some more maybe we shouldn't use a cookie at all. The only reason we use a cookie for login forms is that there's not always a user logged-in. For the admin console though there's always a logged-in user. As the users selected locale is available on the user model we could just add to the whoAmI endpoint to also return the locale? This would be more similar to the approach that users own applications would need to use. They'd add the locale to the token and get it from there.